functions/concepts/pom.xml

```diff
@@ -1,5 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't think you want to remove this. 

@lesv had a comment on a line like this, but I think it was RE: Parent pom and not RE:xml header.

Reply:
See https://github.com/GoogleCloudPlatform/java-docs-samples/pull/2630#discussion_r407596003

i'd say - let's please not remove standard stuff! it's unnecessarily extra work. every pom.xml pretty much has this.

Reply:
We want the `pom.xml` file removed, not the line.

Reply:
Got it - reverted this change for all POMs.